vllm/v1/worker/gpu_model_runner.py

```diff
@@ -1547,7 +1546,8 @@ def get_kv_cache_spec(self) -> KVCacheSpec:
         use_mla = self.vllm_config.model_config.use_mla
         kv_cache_spec: KVCacheSpec = {}
         for layer_name, attn_module in forward_ctx.items():
-            if isinstance(attn_module, FusedMoE):
+            # don't import `FusedMoE` in case triton is not installed
+            if attn_module.__class__.__name__ == "FusedMoE":
```

Comment:
Maybe worth mentioning that this is because the ARM wheels are not available?

Reply:
i find that the attention module also has triton dependency, for `merge_state` . so triton is a hard dependency now.